packages/@react-aria/tooltip/docs/useTooltipTrigger.mdx

```diff
@@ -0,0 +1,132 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tooltip';
+import statelyDocs from 'docs:@react-stately/tooltip';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-aria/switch/package.json';
+
+```jsx import
+import {useTooltipTrigger} from '@react-aria/tooltip';
+import {useTooltip} from '@react-aria/tooltip';
+```
+
+---
+category: Overlays
+keywords: [tooltip, aria]
+---
+
+# useTooltipTrigger
+
+<p>{docs.exports.useTooltipTrigger.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useTooltipTrigger']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-1.2/#tooltip'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useTooltipTrigger} links={docs.links} />
+
+## Features
+
+There is no native HTML element for tooltips. There are many guides out there for
+tooltips, however, they are not easy to make accessible, especially the CSS only ones.
```

Comment:
You could argue the `title` attribute could be used for this...

```suggestion
The HTML `title` attribute can be used to create a tooltip, but it cannot be styled. Custom styled 
tooltips can be hard to build in an accessible way. `useTooltipTrigger` and `useTooltip` help build 
fully accessible tooltips that can be styled as needed.
```

Reply:
good point, i'd forgotten about title since there's almost nothing you can do with it